app/public/templates/tutorial.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>{{ scenario_name }} - Tutorial</title>
+    <script src="https://aframe.io/releases/1.1.0/aframe.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/gh/donmccurdy/aframe-extras@v6.1.1/dist/aframe-extras.controls.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/gh/donmccurdy/aframe-extras@v6.1.1/dist/aframe-extras.pathfinding.min.js"></script>
+    <script src="https://unpkg.com/aframe-environment-component@1.1.0/dist/aframe-environment-component.min.js"></script>
+    <script src="https://unpkg.com/aframe-super-keyboard@2.1.0/dist/aframe-super-keyboard.min.js"></script>
+    <script src="https://unpkg.com/aframe-state-component@7.1.0/dist/aframe-state-component.min.js"></script>
+    <script src="https://unpkg.com/aframe-drag-controls@1.0.0/index.js"></script>
+    <!-- A-frame components -->
+    <!-- TODO: minify and combine all components once we are done -->
+    <script src="/static/js/components/animation-on-click.js"></script>
+    <script src="/static/js/components/close-popup.js"></script>
+    <script src="/static/js/components/keypad.js"></script>
+    <script src="/static/js/components/text-pane.js"></script>
+    <script src="/static/js/components/visual-pane.js"></script>
+    <script src="/static/js/components/hint-button.js"></script>
+    <!-- Initial state -->
+    <script>
+      AFRAME.registerState({
+        initialState: {
+          solvedObjects: {},
+        },
+        handlers: {
+          solvedObject: function (state, action) {
+            state.solvedObjects[action.id] = true;
+          },
+          initializeObject: function (state, action) {
+            state.solvedObjects[action.id] = action.solved;
+          },
+        },
+      });
+    </script>
+  </head>
+  <body>
+    <div id="scene-root">
+      <a-scene id="3-scene" cursor="rayOrigin:mouse" raycaster="objects: .link">
+        <a-assets timeout="50000">
+          <a-asset-item
+            id="1-asset-src"
+            src="{{ asset_prefix_url }}assets/dev/armchair.glb"
```

Comment:
Once you merge latest master in, can you update these to the new file versions of them?
https://github.com/uwblueprint/cdc-backend/blob/master/scripts/mockdata/assets.json

This commit shows the diff: https://github.com/uwblueprint/cdc-backend/pull/116/commits/e4245aa4297c983224810dedfb7200b14302b63d